doc/api/settings.rst

```diff
@@ -0,0 +1,8 @@
+API Documentation: Settings
```

Comment:
Those are not any settings, those are "Platform-specific Settings"

Reply:
Done